site/en/docs/extensions/mv3/intro/mv3-migration/index.md

```diff
@@ -366,122 +445,153 @@ chrome.scripting.executeScript({
-network request, and in most cases needs no host permissions.
+### Can Manifest V3 extensions use blocking Web Request?  {: #when-use-blocking-webrequest }
 
-{% Aside %}
-Request redirects and header modifications **do** require the user to grant host permissions.
-{% endAside %}
+The blocking version of the [Web Request API][api-webrequest] exists in Manifest V3, but it can only be used by extensions that are force-installed using Chrome's Enterprise Policies: 
+[ExtensionSettings][enterprise-settings], [ExtensionInstallForcelist][enterprise-force-list].
 
+Extensions meant to be used by the general public must now use [Declarative Net
+Request][api-declarativenetrequest] for network request modification. Here used by the general
+public means any extension published to the Chrome Web Store except those deployed to a given domain
+or to trusted testers. 
 
 ### How do you use declarativeNetRequest?  {: #how-use-declarativenetrequest }
 
-Instead of reading the request and programmatically altering it, your extension
-specifies a number of rules, which map a set of conditions to corresponding
-actions. See the
-[declarativeNetRequest](/docs/extensions/reference/declarativeNetRequest)
-reference documentation for a more detailed description of rules.
+Instead of reading the request and programmatically altering it, your extension specifies a number
+of [rules][api-declarative-rules]. Each rule contains a set of actions to perform when a given set
+of conditions are matched. For example, you could define a rule that removes "cookie" headers when a
+request is sent to a specific domain. See the [declarativeNetRequest][api-declarativenetrequest]
+reference documentation for a more detailed description of rules
 
-This feature allows content blockers and other request-modifying extensions to
-implement their use cases without requiring host permissions, and without
-needing to read the actual requests.
+This feature allows content blockers and other request-modifying extensions to implement their use
+cases without requiring host permissions, and without needing to read the actual requests.
 
-{% Aside %}
-In order to aid with the migration process, the declarativeNetRequest API is
-available for use in Manifest V2 extensions as of Chrome 84.
-{% endAside %}
+### Conditional permissions and declarativeNetRequest  {: #declarativenetrequest-conditional-perms }
 
+Most use cases for declarativeNetRequest don't require any host permissions at all. However, some
+do.
 
-### Conditional permissions and declarativeNetRequest  {: #declarativenetrequest-conditional-perms }
+{% Aside 'caution' %}
 
-Most use cases for declarativeNetRequest don't require any host permissions at
-all. However, some do.
+Host permissions are still required if the extension wants to *redirect* a request or modify its
+headers. The `declarativeNetRequestWithHostAccess` permission always requires host permissions to
+the request URL and initiator to act on a request.
```

Comment:
Should this be, "to he request URL and its initiator"? (Added the word 'its'.)

Reply:
Yep! Changing now